content/blog/pki-maze.md

```diff
@@ -0,0 +1,72 @@
++++
+title = "Designing a Permanent Personal Identity"
+date = 2019-11-20
+description = "A public key infrastructure (PKI) is a system for binding a set of keys to a name. Sometimes a small amount of metadata is included."
+[extra]
+author = "Philip"
```

Comment:
We tend toward full names on the rest of the blog. Author’s preference of course but don’t we use it elsewhere? In the whitepaper as well?

Reply:
Will email him and get his OK

Reply:
Name OK'd